apps/web-portal-client/lib/components/layout/nav-bar/Navbar.jsx

```diff
@@ -1,13 +1,22 @@
 import React, {useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {userLogout} from '../../../features/auth/effects'
 
 import {Logo} from './Logo'
 import {DesktopMenu} from './DesktopMenu'
-import {LoginButton} from './LoginButton'
-import {MobileMenuButton} from './MobileMenuButton'
+import {Button} from './Button'
+import {Image} from './Image'
+import {imageShapes} from './ImageShapes'
 import {MobileMenu} from './MobileMenu'
 
 export function Navbar() {
-  const [showMobileMenu, setShowMobileMenu] = useState(false)
+  const [shouldShowMobileMenu, setShouldShowMobileMenu] = useState(false)
```

Comment:
`setShouldShowMobileMenu` sounds a bit werid, `toggleMobileMenu` would be better